components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -1187,6 +1187,11 @@ void RewardsServiceImpl::LoadNicewareList(
 void RewardsServiceImpl::SavePublisherInfo(
     ledger::PublisherInfoPtr publisher_info,
     ledger::PublisherInfoCallback callback) {
+  if (!publisher_info) {
```

Comment:
Why are we getting `nullptr` in the first instance, at a minimum we should add a `DCHECK(publisher_info)` so we can track down the culprit